nomad/structs/structs.go

```diff
@@ -5251,6 +5287,89 @@ func (c *Constraint) Validate() error {
 	return mErr.ErrorOrNil()
 }
 
+const (
+	AffinitySetContainsAll = "set_contains_all"
```

Comment:
Why introduce a new `set_contains_all`? Why can't we reuse ConstraintSetContains

Reply:
per our discussion both `set_contains` and set_contains_all` are accepted as equivalent operators.

Reply:
Should we rename to be ConstraintSetContainsAll/Any?